### ISSUE TITLE
This project can be revived

### ISSUE THREAD

**Issue by scj643** [coman3/Google.Music#11]
Using an iOS device that is jailbroken and SSL kill switch one could easily mitm that.

**Comment by coman3** [comment]
Yes it could be revived, but unfortunately i don't have the time any more. There are a few options to decryt the packets such as:  http://www.myhowto.org/java/81-intercepting-and-decrypting-ssl-communications-between-android-phone-and-3rd-party-server/ but again, work as taken priority 😢 sorry!
